src/components/SourcesTree.js

```diff
@@ -17,6 +18,15 @@ const { showMenu } = require("./shared/menu");
 const { copyToTheClipboard } = require("../utils/clipboard");
 const { throttle } = require("../utils/utils");
 
+type CreateTree = {
```

Comment:
I think `any` is ok for now.  This is a good size change and we can dig into that later.